core/services/synchronization/telem/telem_enhanced_ea_mercury.proto

```diff
@@ -59,4 +59,5 @@ message EnhancedEAMercury {
   int64 round=19;
   int64 epoch=20;
   string asset_symbol=21;
+  uint32 don_id=36;
```

Comment:
If we're going to make modifications, may I suggest also adding `string stream_id` and then we can put the feed ID in the `feed` field?